mailbox/store/src/main/java/org/apache/james/mailbox/store/mail/AnnotationMapper.java

```diff
@@ -84,4 +84,20 @@
      */
     void insertAnnotation(MailboxId mailboxId, MailboxAnnotation mailboxAnnotation);
 
+    /**
+     * Checking the current annotation of selected mailbox is existed on store or not.
+     *
+     * @param mailboxId the selected mailbox
+     * @param mailboxAnnotation current annotation should be checked
+     * @return boolean should be 'true' if it already existed on store
+     */
+    boolean exist(MailboxId mailboxId, MailboxAnnotation mailboxAnnotation);
+
+    /**
+     * Getting total number of current annotation on mailbox
+     *
+     * @param mailboxId the selected mailbox
```

Comment:
you can drop @param and  @result when they say the same thing as the method prototype + description